css/main.css

```diff
@@ -0,0 +1 @@
+@import url("style.css");
```

Comment:
So what's special about this is you can create specific css files for specific set of rules. So let's say you have a decent amount of css for a gallery of photos. You can break all that code out into it's own file called `gallery.css` and import it here. 
What's cool about that is now that file is easier to read and also let's say you don't have a gallery on the contact page, will now if or when you create a css file for the contact page, you don't have to include the `gallery.css` file. That's important because you're not making your user download code they're not actually going to use.